src/main/java/org/breedinginsight/brapi/v1/controller/BrapiAuthorizeController.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.breedinginsight.brapi.v1.controller;
+
+import io.micronaut.context.annotation.Property;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.QueryValue;
+import io.micronaut.http.uri.UriBuilder;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.validation.constraints.NotBlank;
+import java.net.URI;
+
+@Slf4j
+@Controller("/${micronaut.brapi.no-version}")
```

Comment:
Why does this have to be read from a property in `application.yml` and not just hard coded as `/brapi`?  I actually have the same question for all of our controllers, but that's outside of the scope of this PR.

Reply:
I believe that pattern came from a micronaut example early on. If it was hardcoded I think would have to be defined in a static variable somewhere because it's used in a lot of different places.

Reply:
Moved from config to code and pushed changes.